framework/vst/devtools/vstdevtools.cpp

```diff
@@ -0,0 +1,155 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+#include "vstdevtools.h"
+#include <QStandardItemModel>
+#include <QStandardItem>
+#include "log.h"
+#include "internal/vstsynthesizer.h"
+
+using namespace mu::vst;
+
+VSTDevTools::VSTDevTools(QObject* parent)
+    : QObject(parent), m_pluginsListModel(new PluginListModel(vstScanner(), this))
+{
+    qmlRegisterType<PluginListModel>("MuseScore.VST", 1, 0, "VSTPluginListModel");
```

Comment:
better register qml types in module setup, method registerUiTypes

Reply:
fixed